package.json

```diff
@@ -20,7 +20,8 @@
     "format:js": "npm run lint:js -- --fix",
     "format:css": "npm run lint:css -- --fix",
     "test": "npm-run-all test:*",
-    "test:php": "wp-env run phpunit 'composer test --working-dir=/var/www/html/wp-content/plugins/two-factor'",
+    "test:php": "npm run composer test",
+    "composer": "wp-env run phpunit composer --working-dir=/var/www/html/wp-content/plugins/two-factor",
```

Comment:
Introduce a new script helper to run any Composer commands inside the dev environment.